tools/mdm/migration/mdmproxy/README.md

```diff
@@ -0,0 +1,28 @@
+Proxy for MDM requests used in seamless migrations, as described in
+https://github.com/fleetdm/fleet/issues/19387.
+
+
+### Usage
+
+```
+Usage of ./mdmproxy:
+  -auth-token string
+        Auth token for remote flag updates (remote updates disabled if not provided)
+  -existing-hostname string
+        Hostname for existing MDM server (eg. 'mdm.example.com') (required)
+  -existing-url string
+        Existing MDM server URL (full path) (required)
+  -fleet-url string
+        Fleet MDM server URL (full path) (required)
+  -migrate-percentage int
+        Percentage of clients to migrate from existing MDM to Fleet
+  -migrate-udids string
+        Comma-delimited list of UDIDs to migrate always
```

Comment:
Is comma delimited true here?

```
robert@Roberts-MacBook-Pro mdmproxy % ./mdmproxy --existing-hostname example.daveherder.com --fleet-url https://micromdm.example.com --migrate-percentage 50  --existing-url https://micromdm.example.com --migrate-udids "a,b,c,d"
2024/06/23 19:07:52 --migrate-udids set: map[a,b,c,d:{}]
2024/06/23 19:07:52 --migrate-percentage set: 50
2024/06/23 19:07:52 --existing-url set: https://micromdm.example.com
2024/06/23 19:07:52 --existing-hostname set: example.daveherder.com
2024/06/23 19:07:52 --fleet-url set: https://micromdm.example.com
2024/06/23 19:07:52 Starting server on :8080
^C
robert@Roberts-MacBook-Pro mdmproxy % ./mdmproxy --existing-hostname example.daveherder.com --fleet-url https://micromdm.example.com --migrate-percentage 50  --existing-url https://micromdm.example.com --migrate-udids "a b c d"
2024/06/23 19:08:02 --migrate-udids set: map[a:{} b:{} c:{} d:{}]
2024/06/23 19:08:02 --migrate-percentage set: 50
2024/06/23 19:08:02 --existing-url set: https://micromdm.example.com
2024/06/23 19:08:02 --existing-hostname set: example.daveherder.com
2024/06/23 19:08:02 --fleet-url set: https://micromdm.example.com
2024/06/23 19:08:02 Starting server on :8080
```
The map doesn't appear to do map things unless split by spaces.

Reply:
Yes, typo! I thought it would be a bit easier to work with if it were newline/space delimited.